cupy/cuda/cudnn.pyx

```diff
@@ -758,7 +758,22 @@ class CuDNNError(RuntimeError):
     def __init__(self, int status):
         self.status = status
         msg = cudnnGetErrorString(<Status>status)
-        super(CuDNNError, self).__init__(msg.decode())
+        super(CuDNNError, self).__init__(
+            'cuDNN Error: {}'.format(msg.decode()))
+        self._infos = []
+
+    def add_info(self, info):
```

Comment:
Where is this routine called?

Reply:
Not called ATM.
There should be a test, but I'm not sure how it can be tested.